pkg/addon/utils.go

```diff
@@ -52,12 +52,17 @@ const (
 	traitMapKey               = "trait"
 	wfStepMapKey              = "wfStep"
 	policyMapKey              = "policy"
+	annotationBindCompKey     = "addon.oam.dev/ignore-without-component"
```

Comment:
```suggestion        
	annotationBindCompKey     = "addon.oam.dev/bind-component"
```

Move it along with other annotations